### DISCUSSION TITLE
Cannot seem to get it working correctly with motion sensor.

### DISCUSSION THREAD

**Discussion by asinla** [basnijholt/adaptive-lighting#948] [Q&A]
I'm sure I cannot be the only one with this issue but I could not find any answers.

AL is setup with the kitchen lights which go on with a motion sensor. The lights come on at the correct brightness in the evening/night but then if the motion sensor gets triggered again after it's timeout the light shoot up to full brightness. I have tried all settings in the config relating to Takeover Control but those settings don't seem to have any effect on anything.

Any help greatly appreciated.

Thanks.

-A

**Comment by asinla** [comment]
No one? 

Can someone direct me to a post that explains it better?

Thanks

**Comment by asinla** [comment]
Please??!

**Comment by basnijholt** [comment]
This is most likely a hardware issue and not something that AL can fix.

**Reply by asinla** [reply]
Thanks for the reply.

It is an Aquara motion sensor, quite common. Are you saying that the motion sensor is the issue? It happens on all lights that use motion sensors.
What is the correct combination of config settings to allow the motion sensor to turn on the lights with the AL settings and then keep those AL settings even if the motion sensor is triggered again with the lights still on?

Cheers.

**Reply by basnijholt** [reply]
I meant the light. The motion sensor should be unrelated.

**Reply by asinla** [reply]
Thanks again.

Not sure how the light is the problem?  I am using a combination of Lutron Caseta lightswitches and Philips Hue lights. 

I have the same issue with whatever light I use. When AL is used on any lights with a motion sensor I have this problem. It seems AL is ignoring the instructions to keep adapting the lights even if a new light_on command is received from the motion sensor.

Again, from my previous post.....

What is the correct combination of config settings to allow the motion sensor to turn on the lights with the AL settings and then keep those AL settings even if the motion sensor is triggered again with the lights still on?

Thanks so much for your time. I really hope to get this sorted out.

-A

**Reply by asinla** [reply]
Did I say something wrong?
